conf/test_full.config

```diff
@@ -4,7 +4,7 @@
  * -------------------------------------------------
  * Defines bundled input files and everything required
  * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/proteomicslfq -profile test,<docker/singularity>
+ *   nextflow run nf-core/proteomicslfq -profile test_full,<docker/singularity>
```

Comment:
Well that ain't really a fast test, is it?

I would suggest to keep the line and add a second line like:

> For a more comprehensive test you can run 
> 
> `nextflow run nf-core/proteomicslfq -profile test_full,<docker/singularity>`
> 
>

Reply:
I opted for a mention of the basic tests only. Since the actual command is listed in the corresponding test.config file.